activerecord/lib/active_record/relation/calculations.rb

```diff
@@ -343,14 +343,15 @@ def execute_grouped_calculation(operation, column_name, distinct) # :nodoc:
         group_columns = group_aliases.zip(group_fields)
 
         column = aggregate_column(column_name)
-        column_alias = column_alias_for("#{operation} #{column_name.to_s.downcase}")
+        column_alias = column_alias_for("#{operation}_#{column_name.to_s.downcase}")
```

Comment:
My intent for using the space is to squash multiple _ in the alias. Is there any worth replacing the space to _ here?

https://github.com/rails/rails/blob/ec7f0580b3421f2ca42c22b16c79eadcaf66c485/activerecord/lib/active_record/relation/calculations.rb#L411-L413

Reply:
My bad, I thought it was a waste to use `" "` when it would knowingly be changed for `"_"` by gsub, but it's totally out of scope here I shouldn't  have make this change anyway.